src/utils.js

```diff
@@ -20,7 +20,6 @@ const checkFilePath = async (path) => {
  * @param {string} folderNamePath
  *  */
 const isDirectory = (folderNamePath) => {
-  folderNamePath = fixPath(folderNamePath)
```

Comment:
are you sure that removing fixPath is a good thing to do?

Reply:
> next time please fill a PR description template, so it'll be easy for me to review your changes
> 
> #146
I was unaware of this. I'll make sure I do that from next time.

Also, the fixPath line was something I added, and it gave rise to a few errors, so was testing it out.

Can you please guide me on how to check if the code is correct locally?

Reply:
ok, got it - you add a fix path line. just trying to catch all small issues.

right now you can test only if build is passed by running `yarn build` or `yarn build:babel`

in order to have more advanced coverage - we need to make more tests and also fix eslint warnings so our travis will will be able to pass